packages/auth/src/core/auth/emulator.ts

```diff
@@ -97,13 +102,14 @@ export function connectAuthEmulator(
   authInternal.emulatorConfig = emulatorConfig;
   authInternal.settings.appVerificationDisabledForTesting = true;
 
-  if (!disableWarnings) {
+  if (!disableWarnings && !isCloudWorkstation(host)) {
     emitEmulatorWarning();
   }
 
   // Workaround to get cookies in Firebase Studio
   if (isCloudWorkstation(host)) {
     void pingServer(`${protocol}//${host}${portStr}`);
+    updateEmulatorBanner('Auth', true);
```

Comment:
should we honor the disableWarnings flag for this banner? I think the way you have it now there's no way for customer to override it